src/main/resources/db/migration/common/V1_210__insert_transcription_status_data.sql

```diff
@@ -0,0 +1,6 @@
+INSERT INTO darts.transcription_status VALUES(1, 'Requested');
+INSERT INTO darts.transcription_status VALUES(2, 'Awaiting Authorisation');
+INSERT INTO darts.transcription_status VALUES(3, 'With Transcriber');
+INSERT INTO darts.transcription_status VALUES(4, 'Complete');
+INSERT INTO darts.transcription_status VALUES(5, 'Rejected');
+INSERT INTO darts.transcription_status VALUES(6, 'Closed');
```

Comment:
I guess we need to restart the sequence `trs_seq` to be next value 7 - since we have one!
Even though I don't think we really need to have the sequence for a standing data table